ci_scripts/docker-coredeps/deps/run_all.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+#
+# Build the Delta Chat C/Rust library
+#
+set -e -x
+
+# perform clean build of core and install 
+export NINJA_BUILD_DIR=.docker-corebuild
```

Comment:
A hidden directory is a rather weird choice here

Reply:
i don't want to interfere with the host system -- all this scripts run in docker and write to the host system, for now.

Reply:
you're still writing to the host though, only the user doesn't see it.  if you don't want to interfere then maybe check if the directory exists and if it does bail out with an error on the CI it shouldn't exist so will work?

Reply:
i am not up for doing the circle-ci cycles right now and suggest to do this later. In my testing and usage it seems to be fine.  We could use more of circle-ci to isolate things, i.e. not mount from the host -- i think circle-ci does let you start in the checkout but this checkout is a layer, not a direct mount. I might be mistaken.  However, that would mean not being able to run things locally and for now i prefer to be able to have quick cycles locally.  This spurious artifact is only a minor annoyance IMO.